application/client/tsconfig.json

```diff
@@ -41,6 +41,7 @@
       "@tabs/*": ["src/app/ui/tabs/*"],
       "@statusbar/*": ["src/app/ui/statusbar/*"],
       "@sidebar/*": ["src/app/ui/sidebar/*"],
+      "@wasm/*": ["../apps/webassembly/*"],
```

Comment:
As soon as we will have multiple wasm-module and probably in folder `/apps/webassembly` we will also have some support/help folders (like `/app/webassembly/testing` to store common test-related code), it's better to use direct paths:
```"@matcher/*": ["../apps/webassembly/matcher/pkg/*"]```